dtool/Config.cmake

```diff
@@ -0,0 +1,629 @@
+
+set(PRC_EXECUTABLE_ARGS_ENVVAR "PANDA_PRC_XARGS" CACHE STRING
+  "The environment variable that defines optional args to pass to
+executables found that match one of the above patterns.")
+
+# You can implement signed prc files, if you require this advanced
+# feature.  This allows certain config variables to be set only by a
+# prc file that has been provided by a trusted source.  To do this,
+# first install and compile Dtool with OpenSSL and run the program
+# make-prc-key, and then specify here the output filename generated
+# by that program, and then recompile Dtool.
+
+set(PRC_PUBLIC_KEYS_FILENAME "" CACHE STRING "")
+
+# By default, the signed-prc feature, above, is enabled only for a
+# release build.  In a normal development environment, any prc file
+# can set any config variable, whether  or not it is signed.  Set
+# this variable true or false to explicitly enable or disable this
+# feature.
+#XXX For which build types should this be enabled?
+if(CMAKE_BUILD_TYPE STREQUAL "Release")
+  set(DEFAULT_PRC_RESPECT_TRUST_LEVEL ON)
+else()
+  set(DEFAULT_PRC_RESPECT_TRUST_LEVEL OFF)
+endif()
+
+option(PRC_RESPECT_TRUST_LEVEL
+  "Define if we want to enable the trust_level feature of prc config
+variables.  This requires OpenSSL and PRC_PUBLIC_KEYS_FILENAME,
+above." ${DEFAULT_PRC_RESPECT_TRUST_LEVEL})
+
+# If trust level is in effect, this specifies the default trust level
+# for any legacy (Dconfig) config variables (that is, variables
+# created using the config.GetBool(), etc. interface, rather than the
+# newer ConfigVariableBool interface).
+
+set(PRC_DCONFIG_TRUST_LEVEL "0" CACHE STRING
+  "The trust level value for any legacy (DConfig) variables.")
+
+# If trust level is in effect, you may globally increment the
+# (mis)trust level of all variables by the specified amount.
+# Incrementing this value by 1 will cause all variables to require at
+# least a level 1 signature.
+
+set(PRC_INC_TRUST_LEVEL "0" CACHE STRING
+  "The amount by which we globally increment the trust level.")
+
+# Similarly, the descriptions are normally saved only in a
+# development build, not in a release build.  Set this value true to
+# explicitly save them anyway.
+#XXX only for release-release builds
+option(PRC_SAVE_DESCRIPTIONS
+  "Define if you want to save the descriptions for ConfigVariables."
+  ON)
+
+mark_as_advanced(DEFAULT_PRC_DIR PRC_DIR_ENVVARS PRC_PATH_ENVVARS
+  PRC_PATTERNS PRC_ENCRYPTED_PATTERNS PRC_ENCRYPTION_KEY
+  PRC_EXECUTABLE_PATTERNS PRC_EXECUTABLE_ARGS_ENVVAR
+  PRC_PUBLIC_KEYS_FILENAME PRC_RESPECT_TRUST_LEVEL
+  PRC_DCONFIG_TRUST_LEVEL PRC_INC_TRUST_LEVEL PRC_SAVE_DESCRIPTIONS)
+
+#
+# This is the end of the PRC variable customization section.  The
+# remaining variables are of general interest to everyone.
+#
+
+
+option(HAVE_P3D_RTDIST
+  "You may define this to build or develop the Panda3D rtdist,
+the environment packaged up for distribution with the plugin."
+  OFF)
```

Comment:
As far as I'm concerned we can axe the rtdist.  If CMake becomes its grave, I shall not weep.

Reply:
Good because that isn't tested at all.